### DISCUSSION TITLE
Multi-level menus

### DISCUSSION THREAD

**Discussion by kla-ko** [GeorgianStan/vanilla-context-menu#32] [Ideas & Suggestions]
I like this library very much. Easy to use and customize.
The only thing I'm missing is the capability to create multi-level menus.
Any chance that this might be added at some point in time?

Regards,

**Comment by GeorgianStan** [comment]
Hi @kla-ko! The intention is to let users decide the development roadmap. If multi-level menu is a requested feature, then yes we'll add it.

**Comment by fab032** [comment]
Hi, I also really like this library. And I'm also interested in creating multi-levels menus.
Have you already planned the development of the feature?
Regards

**Comment by GeorgianStan** [comment]
Hi @fab032 . I've linked this library to a [project](https://github.com/GeorgianStan/vanilla-context-menu/projects?query=is%3Aopen) and I'm planning to do it soon. It's second in list.

**Reply by fab032** [reply]
OK, great news! Thank you for your prompt response

**Reply by kla-ko** [reply]
Good news. I will be happy to test any early version ...

**Reply by GeorgianStan** [reply]
Hi @kla-ko! I drafted a new release candidate with the nested menu(one level only) feature. [Here](https://github.com/GeorgianStan/vanilla-context-menu/discussions/34) is the related discussion page. 
The RC can be downloaded and tested. Any feedback is welcome. :).

**Reply by kla-ko** [reply]
Hi @GeorgianStan ,
Thanks a lot. I can look at this next week latest ..
